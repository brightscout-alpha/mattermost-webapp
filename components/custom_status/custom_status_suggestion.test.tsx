// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.
import {shallow} from 'enzyme';
import React from 'react';

import CustomStatusSuggestion from './custom_status_suggestion';

describe('components/custom_status/custom_status_emoji', () => {
    const baseProps = {
        handleSuggestionClick: jest.fn(),
        emoji: '',
        text: '',
        handleClear: jest.fn(),
    };

    it('should match snapshot', () => {
        const wrapper = shallow(
            <CustomStatusSuggestion {...baseProps}/>,
        );

        expect(wrapper).toMatchSnapshot();
    });
});
