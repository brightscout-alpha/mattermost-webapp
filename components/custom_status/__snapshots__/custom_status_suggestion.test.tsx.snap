// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/custom_status/custom_status_emoji should match snapshot 1`] = `
<div
  className="statusSuggestion__row cursor--pointer a11y--active"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="statusSuggestion__icon"
  >
    <RenderEmoji
      emoji=""
      size={20}
    />
  </div>
  <span
    className="statusSuggestion__text"
  />
</div>
`;
